templates/base.html

```diff
@@ -74,6 +74,7 @@
         <meta name="msapplication-TileImage" content="{{ STATIC_URL }}mstile-144x144.png">
     </head>
     <body class="{% if profile %}{% if is_external_student %}external-student{% else %}internal-student{% endif %}{% endif %} lang-{{ request.LANGUAGE_CODE }}" data-view-tag="{% block view_tag %}{% endblock %}">
+        <a href="#content" class="skip-link" aria-label="{% trans "Main navigation" %}">{% trans "Skip main navigation" %}</a>
```

Comment:
I think I'm just unknowledged about this, but should the `aria-label` be in the element that contains the labeled part, thus in this case in the `<nav -class="topbar"`? I assume it doesn't matter so much as the main importance is that the screen reader reads it at the correct time. Similarly, should this link be as the first thing within the topbar nav or just before that nav as a sibling element? I'm not familiar enough to know this, thus I'm mainly interested for how this location was selected assuming there are some reason for it.

Reply:
Quite right @raphendyr , I copy/pasted the `aria-label` into the wrong element between testing it and fixing the translations. It's fixed now. 

Generally, the most important thing is that it's the first element that can receive keyboard focus, which is why it's not nested inside the main navigation. I also like the visual effect when it sticks to the top of the page and displaces the content a little bit.

Reply:
> I also like the visual effect when it sticks to the top of the page and displaces all of the content a little bit.

I like it too. I didn't remember that, when I was thinking the location of the element.